CONTRIBUTING.md

```diff
@@ -1,26 +1,101 @@
-# Dynatrace Operator
+- [Contributing](#contributing)
+  - [How to run unit tests](#how-to-run-unit-tests)
+  - [How to run E2E tests](#how-to-run-e2e-tests)
+  - [Coding style guide](#coding-style-guide)
```

Comment:
It makes sense to move the code style to its own document, so we can easily extend it and polish it.

Reply:
Good idea! Done :+1: